cmake/ElektraCache.cmake

```diff
@@ -37,7 +37,7 @@ set (ADDED_DIRECTORIES "" CACHE STRING ${PLUGINS_DOC} FORCE)
 
 remember_for_removal(BINDINGS TO_REMOVE_BINDINGS)
 
-set (BINDINGS_LIST_DEFAULT cpp)
+set (BINDINGS_LIST_DEFAULT cpp;glib;gsettings)
```

Comment:
This was just to get the tests running. I would like to postpone adding the module to ALL till it is more stable.